garage/tf/policies/deterministic_mlp_policy_with_model.py

```diff
@@ -96,9 +96,12 @@ def get_action_sym(self, obs_var, name=None, **kwargs):
         """Return action sym according to obs_var."""
         with tf.variable_scope(self._variable_scope):
             action = self.model.build(obs_var, name=name)
-            action = tf.reshape(action, self.action_space.shape)
             return action
 
+    def get_params(self, trainable=True):
+        """Get the trainable variables."""
+        return self.get_trainable_vars()
+
```

Comment:
I think it's time to move `get_params` into base class. Please also update all other policies that have `get_params`.